### PR TITLE
Fs 3281 refactor flag endpoints

### PR DESCRIPTION
**Pull request by Adam-W1**
https://dluhcdigital.atlassian.net/browse/FS-3281

### Change description
- Old flagging endpoint is obsolete now. Once we archived the single old flag application histories(FS-3008).  We need to remove old flagging endpoints, functions & update tests.

- Remove old flag endpoints, functions and tests.

- Migration script to drop old flag tables 

Make sure unit tests are updated to flags_v2

- [x] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
pytest

### Screenshots of UI changes (if applicable)
image

### CODE REVIEW

**Review by RamuniN — APPROVED** [review]
Nice work in tidying up.  I can see the effort in refactoring the code. LGTM 👍

**Review by RamuniN — CHANGES_REQUESTED** [review]
Noticed this error when ran the `flask db downgrade 55dcd4abf66a`
image
Can you check if the downgrade function is working fine ?

File: db/migrations/versions/222a1c3b6321_.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+revision = "222a1c3b6321"
+down_revision = "55dcd4abf66a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("flags")
+
```
Comment (RamuniN) [inline db/migrations/versions/222a1c3b6321_.py:22]:
```suggestion
op.execute("DROP TYPE flagtype")
```

`flagtype` is not automatically dropped. This line should fix the previously mentioned error

**Review by RamuniN — APPROVED** [review]
👍

File: db/migrations/versions/222a1c3b6321_.py (hunk 1 above)
Comment (Adam-W1) [reply db/migrations/versions/222a1c3b6321_.py:23]: thanks @RamuniN

**Pull request merged**